### ISSUE TITLE
Bad Substitution (nested variable access)

### ISSUE THREAD

**Issue by raabf** [simonwhitaker/gibo#39]
When using `gibo dump` with gibo 2.2.1, the following error occurs:

    ~/bin/gibo: 95: ~/bin/gibo: Bad substitution

https://github.com/simonwhitaker/gibo/blob/61d0aff95baf0d1efc361ef69c6a37f71913a894/gibo#L95

Indeed, the variable access is here nested, I think this should be:

    echo "### $remote_web_root${language_file}/$local_repo/"

**Comment by simonwhitaker** [comment]
Good catch! That line only works on Bash. It looks like the non-Bash-specific way to do this is `${language_file#"$local_repo/"}`. Fix on the way!

**Comment by raabf** [comment]
Thank you! I never saw this syntax before.
I tested your fix and of course it works (also with zsh), but just for interest for the old syntax:
Indeed I use zsh, but I also tested bash (and just now again to be sure) and it also complains about `95: ~/bin/gibo: Bad substitution`. This is not a new feature of bash? (or a deprecated?) (I am using GNU bash, Version 4.4.19(1)-release (x86_64-pc-linux-gnu) from Ubuntu 18.04) Is there a name for this kind of function?
